packages/sdks/javascript/src/api/generated/models/virtual-host-relation.ts

```diff
@@ -27,29 +30,41 @@ export interface VirtualHostRelation {
      */
     id: number;
     /**
-     * The type of the Virtual Host (`name-based` or `ip-based`)
-     * @type {string}
+     * 
+     * @type {VirtualHostType}
      * @memberof VirtualHostRelation
      */
-    type: string;
+    type: VirtualHostType;
     /**
      * The name of the Virtual Host
      * @type {string}
      * @memberof VirtualHostRelation
      */
     name: string;
     /**
-     * The relative path to the root directory of the Virtual Host
+     * 
+     * @type {ApplicationRelation}
+     * @memberof VirtualHostRelation
+     */
+    application: ApplicationRelation | null;
+    /**
+     * The document root location, within the application directory, that contains the public files to be served when a user visits the domain name associated with this virtual host
      * @type {string}
      * @memberof VirtualHostRelation
      */
     root_directory: string;
```

Comment:
@Diegiwg shouldn't this field include null? I believe a virtual host is not required to have a `root_directory`

```suggestion
    root_directory: string | null;
```

Reply:
As seen in VirtualHostCreate and VirtualHostUpdate

https://github.com/devopness/devopness/blob/bdd720bc1b986323abc89ab7013718891ee6ee43/packages/sdks/javascript/src/api/generated/models/virtual-host-environment-create.ts#L39

https://github.com/devopness/devopness/blob/bdd720bc1b986323abc89ab7013718891ee6ee43/packages/sdks/javascript/src/api/generated/models/virtual-host-update.ts#L39